tests/integration_tests/configs/cosmovisor.jsonnet

```diff
@@ -2,13 +2,14 @@ local config = import 'default.jsonnet';
 
 config {
   'ethermint_9000-1'+: {
+    'app-config'+: {
+      'minimum-gas-prices': '100000000000aphoton',
+    },
     genesis+: {
       app_state+: {
         feemarket+: {
           params+: {
-            no_base_fee: false,
             base_fee:: super.base_fee,
-            initial_base_fee: super.base_fee,
```

Comment:
we don't need these anymore?

Reply:
no, based on genesis state validation in new version